src/app/ui/team-page/team-members/team-members.component.ts

```diff
@@ -50,7 +54,10 @@ export class TeamMembersComponent implements OnInit {
 
     this.confirmationModal.showConfirmation({ message }).subscribe(result => {
       if (result) {
-        this.teamsService.removeTeamMember(uid);
+        this.userService.getUser(uid).subscribe(usr => {
```

Comment:
i think we can just pass in the member object from the template and have `removeTeamMember(member: Membership)` here so that we don't have to query users collection to get these info.